### ISSUE TITLE
Warnings / Error Messages creating / loading a project

### ISSUE THREAD

**Issue by haansi** [Integrative-Transcriptomics/MitoBench#65]
When importing data, it would be great to have a warning / error message if a project is already open. 
One has to create a new project to do so.  
Click on "New Project" does not ask if the current project should be saved. 

Hope you can understand what I mean, otherwise please let me know. Many thanks in advance!!!

**Comment by JudithNeukamm** [comment]
Yes, it's clear. Thanks for that, I will include it in the next version.
